frontend/src/components/ButtonSubmit.vue

```diff
@@ -14,31 +14,31 @@
 </template>
 
 <script>
-  /**
+/**
    * A button submit component.
```

Comment:
It seems you've forgot to insertion the comments. It's the same issue in all the components.
```suggestion
/**
  * A button submit component.
  */
```